### ISSUE TITLE
Add bill title, number, description

### ISSUE THREAD

**Issue by wlabranche** [omnibus-app/Omnibus#115]
Include these in the individual bill displays

**Comment by bttmly** [comment]
Care to elaborate? Which view

**Comment by wlabranche** [comment]
Up for discussion, there isn't any clear identifying information about the bill being shown. I think the bill number, name, and maybe description should be clearly presented somewhere.

**Comment by bttmly** [comment]
In the bar chart view you mean? If yes then I agree

**Comment by MSchip** [comment]
I haven't been able to come up with a very good way.  I'm putting in a hack
right now.. but it's only meant for tomorrow.

**Comment by bttmly** [comment]
The donut chart in the bar chart view, that's referring to the "parent" bill right, not the amendment?

**Comment by bttmly** [comment]
I think maybe a strip between the search and the bar chart would be good. bill name and id

**Comment by MSchip** [comment]
Correct. It represents an aggregation of all of the amendment [sic] votes

**Comment by bttmly** [comment]
Oh, interesting. Could we just have it be the same thing but for the hovered amendment?

**Comment by MSchip** [comment]
I don't think all amendments have vote objects
